src/main/java/no/nav/veilarbveileder/service/AuditLogService.java

```diff
@@ -0,0 +1,26 @@
+package no.nav.veilarbveileder.service;
+
+import no.nav.common.audit_log.cef.CefMessage;
+import no.nav.common.audit_log.cef.CefMessageEvent;
+import no.nav.common.audit_log.cef.CefMessageSeverity;
+import no.nav.common.audit_log.log.AuditLogger;
+import no.nav.common.audit_log.log.AuditLoggerImpl;
+import org.springframework.stereotype.Service;
+
+@Service
+public class AuditLogService {
```

Comment:
brukes dette noe sted nå?

Må tas i bruk der hvor det gjøres et kall mot poao-tilgang :)

Reply:
Tror isteden for å lage en AuditLogService ville jeg heller bare i ApplicationConfig.kt lagt inn en bønne: 

`@Bean
    fun auditLogger(): AuditLogger {
        return AuditLoggerImpl()
    }`

for så å ta det inn i AuthService.kt `private val auditLogger: AuditLogger`
og så lage en funksjon av det du har inni her og så kalle den hver gang du gjør et authkall. Om det gir mening. Tar gjerne en huddle på det :)